pyramid_oereb/contrib/print_proxy/mapfish_print.py

```diff
@@ -117,6 +117,19 @@ def __call__(self, value, system):
             }
             restriction_on_landownership['legend'] = restriction_on_landownership['Map'].get(
                 'LegendAtWeb', '')
+
+            # Legend of  other visible restriction objects in the topic map
+            restriction_on_landownership['OtherLegend'] = restriction_on_landownership['Map'].get(
+                'OtherLegend', [])
+            if restriction_on_landownership['OtherLegend'] != '':
```

Comment:
Does it happens? (to be false)

Reply:
Forgot that one, sorry. 🙈 
It just happens to be None sometimes